samples/nrf9160/modem_shell/src/link/link_shell.c

```diff
@@ -463,13 +463,15 @@ static struct option long_options[] = {
 	{ 0, 0, 0, 0 }
 };
 
+static const char short_options[] = "a:I:f:i:x:w:p:t:A:P:U:su014rmngMNed";
+
 /******************************************************************************/
 
 bool link_shell_msleep_notifications_subscribed;
 
-static void link_shell_print_usage(struct link_shell_cmd_args_t *link_cmd_args)
+static void link_shell_print_usage(enum link_shell_command command)
 {
-	switch (link_cmd_args->command) {
+	switch (command) {
```

Comment:
Currently `connect`/`disconnect` commands print usage for both commands. The reasoning has probably been that the commands are related. If we want to group those together, what about moving both under a new subcommand, for example `apn` or `pdn`?

Reply:
I've been thinking about splitting these help texts and I've done that now